metrics/prometheus/prometheus.go

```diff
@@ -51,6 +51,9 @@ type Metrics struct {
 	privacyCOPPA                 *prometheus.CounterVec
 	privacyLMT                   *prometheus.CounterVec
 	privacyTCF                   *prometheus.CounterVec
+	storedResponses              prometheus.Counter
+	storedResponsesFetchTimer    *prometheus.HistogramVec
+	storedResponsesErrors        *prometheus.CounterVec
```

Comment:
Does go_metrics also have the fetch timer and errors metrics? Ideally, we should keep parity between the two.

Reply:
Yes, I believe these two from go_metrics
```
StoredDataFetchTimer           map[StoredDataType]map[StoredDataFetchType]metrics.Timer
StoredDataErrorMeter           map[StoredDataType]map[StoredDataError]metrics.Meter
```
are metrics similar to 
```
storedResponsesFetchTimer    *prometheus.HistogramVec
storedResponsesErrors        *prometheus.CounterVec
```
from prometheus.go. 
In both cases they work using existing functionality to track stored data. 

go_metrics -> 
StoredResponsesMeter  metrics.Meter 
is similar to 
prometheus.go -> storedResponses  prometheus.Counter